src/meta/processors/admin/AdminClient.cpp

```diff
@@ -589,6 +613,10 @@ folly::Future<Status> AdminClient::createSnapshot(GraphSpaceID spaceId, const st
 folly::Future<Status> AdminClient::dropSnapshot(GraphSpaceID spaceId,
                                                 const std::string& name,
                                                 const std::vector<HostAddr>& hosts) {
+    if (injector_) {
+        return injector_->dropSnapshot();
```

Comment:
Do you really need the injector_?  It is used to test the logic for balance.

Reply:
Support some test case in `AdminClientTest`

Reply:
Fine.